app/_components/PrimaryButton/PrimaryButton.tsx

```diff
@@ -0,0 +1,35 @@
+import Link from 'next/link';
+import React from 'react';
+
+type PrimaryButtonPropType = {
+  to: string;
+  size: string;
+  buttonText: string;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+const PrimaryButton = ({
+  to,
+  size,
+  buttonText,
+  onClick,
+}: PrimaryButtonPropType) => {
+  const { width, height } = {
+    width: size === 'large' ? 'w-80' : 'w-44',
+    height: size === 'large' ? 'h-12' : 'h-10',
+  };
```

Comment:
저희 디자인 시스템 스터디할 때 코드 컨벤션을 적용해보면 좋을 것 같아요..!
컴포넌트화를 시키니 `tw-merge`도 설치할 필요가 있을 것 같네요.

```suggestion
const primaryButtonSizes = {
    small: "w-44 h-10",
    large: "w-80 h-12",
  };
  ...
  className={`${primaryButtonSizes[size]} rounded-md bg-[#FFE429] text-[black]`}
```